packages/lb-annotation/src/utils/tool/MarkerUtils.ts

```diff
@@ -1,5 +1,16 @@
+import { cloneDeep } from 'lodash';
 import AttributeUtils from './AttributeUtils';
 
+declare interface IAuxiliaryLine {
+  start: string;
+  end: string;
+}
+
+declare interface IAuxiliaryLineCoord {
```

Comment:
What is difference between IAuxiliaryLine and IAuxiliaryLineCoord？

Reply:
I want to change `IAuxiliaryLine` this way
```typescript
declare interface IAuxiliaryLine {
  start: IPointUnit['label'];
  end: IPointUnit['label'];
}
```

Reply:
Thanks for your description~